domains/kevin.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
There is a conflict with the subdomain you are trying to register as another `kevin.json` was just added to the repo. You will have to change the name of the file.